### DISCUSSION TITLE
Park UI vs Ark UI – what's the difference?

### DISCUSSION THREAD

**Discussion by punkpeye** [chakra-ui/ark#3067] [General]
Looks like Park UI is built on top or Ark, but as far as I can tell, they visually look very similar. So what's the difference?

**Comment by Fazla-GroM** [comment]
Hey @punkpeye 👋 
Ark UI is headless/unstyled by default. It provides unstyled components which handle ui logic required for component to work properly.  Park UI takes Ark UI and just applies theme to it.

The reason why they have similar look and feel websites is beacuse Ark UI website uses Park UI theme 😄  as you can see [here](https://github.com/chakra-ui/ark/blob/main/website/panda.config.ts)

**Reply by punkpeye** [reply]
@Fazla-GroM That makes sense. So I see that Ark has premium features – that I think are really cool. When will Park have those?

**Reply by cschroeter** [reply]
Just a matter of doing it. They are already implemented in Park. Just need more time to port them

**Reply by punkpeye** [reply]
For what's it worth, I would totally pay for it.
